client/lib/geocoding/test/index.js

```diff
@@ -22,37 +17,27 @@ describe( 'geocoding', () => {
 	describe( 'when converting a search string to location results', () => {
 		describe( '#geocode()', () => {
 			test( 'should return a promise', () => {
-				expect( geocode( TEST_ADDRESS ) ).to.be.an.instanceof( Promise );
+				expect( geocode( TEST_ADDRESS ) ).toBeInstanceOf( Promise );
 			} );
 
-			test( 'should call to the Google Maps API', () => {
-				return new Promise( ( done ) => {
-					geocode( TEST_ADDRESS )
-						.then( ( results ) => {
-							expect( results ).to.eql( [ 1, 2, 3 ] );
-							done();
-						} )
-						.catch( done );
```

Comment:
Using `done` as the `catch` handler caused the test to pass even if `geocode` throws. This test wasn't actually testing anything.